run.sh

```diff
@@ -31,6 +31,7 @@ fi
 if [ $SYSTEMTESTS_SCENARIO = "DEFAULT" ]; then  # Most common use case
     export RUNNER_ARGS=tests/
     export SYSTEMTESTS_LOG_FOLDER=logs
+    CONTAINERS+=(postgres)
```

Comment:
Is it mandatory for the test ? the issue is it adding this service to the default scenario adds a big overhead for lot of CI.

If tracer job happen before than the PG client realize  that the service does not respond, it can do the trick, by simply ignoring exception raised.

Reply:
It is necessary to test SQL injection in nodejs.

Although in java they can test it with in a memory database, in nodejs we can't do it because we nodejs library does not intercept something like JDBC driver. We have to detect vulnerability in the database library (pg, mysql...), it means that we need the database to test it.